src/Catalyst.Core.Modules.Ledger/Ledger.cs

```diff
@@ -56,28 +56,39 @@ public sealed class Ledger : ILedger, IDisposable
         private readonly IStateProvider _stateProvider;
         private readonly IStorageProvider _storageProvider;
         private readonly ISnapshotableDb _stateDb;
-        private readonly ISnapshotableDb _codeDb;
+        private readonly IDb _codeDb;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_codeDb' or refactor the code to use its value.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4979613)